creating_packages/define_abi_compatibility.rst

```diff
@@ -226,6 +227,88 @@ the ``optimized`` option to conditionally require different dependencies, that w
+the ``Visual Studio`` compiler in Windows environments and ``gcc`` in Linux environments.
+
+The ``intel`` compiler is declared this way in the :ref:`settings.yml<settings_yml>`:
+
+ .. code-block:: yaml
+
+    intel:
+        version: ["11", "12", "13", "14", "15", "16", "17", "18", "19"]
+        base:
+            gcc:
+                <<: *gcc
+                threads: [None]
+                exception: [None]
+            Visual Studio:
+                <<: *visual_studio
+
+Remember, you can :ref:`extend Conan<extending>` to support other compilers.
+
+
+You can use the ``package_id()`` method to define the compatibility between the packages generated by the ``base`` compiler and the ``parent`` one.
+You can use the following helpers together with the :ref:`compatible packages<compatible_packages>` feature to:
+
+    - Consume native ``Visual Studio`` packages when the input compiler in the profile is ``intel`` (if no ``intel`` package is available).
+    - The opposite, consume an ``intel`` compiler package when a consumer profile specifies ``Visual Studio`` as the input compiler (if no
+      ``Visual Studio`` package is available).
+
+- ``base_compatible()``: This function will transform the settings used to calculate the package ID into the "base" compiler.
+
+  .. code-block:: python
+
+      def package_id(self):
+
+          if self.settings.compiler == "intel":
+              p = self.info.clone()
+              p.base_compatible()
+              self.compatible_packages.append(p)
+
+  Using the above ``package_id()`` method, if a consumer specifies a profile with a intel profile (**-s compiler=="intel"**) and there is no binary available, it will resolve to a
+  Visual Studio package ID corresponding to the base compiler.
+
+
+- ``parent_compatible(compiler="compiler", version="version")``: This function transforms the settings of a compiler into the settings of a
+  parent one using the specified one as the base compiler. As the details of the "parent" compatible cannot be guessed, you have to provide them as **keyword args** to the
+  function. The "compiler" argument is mandatory, the rest of keyword arguments will be used to initialize the ``info.settings.compiler.XXX`` objects
+  to calculate the correct package ID.
+
+  .. code-block:: python
+
+      def package_id(self):
+
+         if self.settings.compiler == "Visual Studio":
+            compatible_pkg = self.info.clone()
+            compatible_pkg.parent_compatible(compiler="intel", version=16)
+            self.compatible_packages.append(compatible_pkg)
+
+  In this case, for a consumer specifying Visual Studio compiler, if no package is found, it will search for an "intel" package for the version 16.
+
+
+Take into account that you can use also this helpers without the "compatible packages" feature:
+
+  .. code-block:: python
+
+      def package_id(self):
+
+         if self.settings.compiler == "Visual Studio":
```

Comment:
I am not sure about the value of this example, I am thinking it could be omitted at the moment.

Reply:
I think it helps to understand the helpers and the compatible packages.